flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -3917,8 +3968,8 @@ static void genOMP(lower::AbstractConverter &converter, lower::SymMap &symTable,
           parser::omp::GetOmpDirectiveName(*ompNestedLoopCons).v;
       switch (nestedDirective) {
       case llvm::omp::Directive::OMPD_tile:
-        // Emit the omp.loop_nest with annotation for tiling
-        genOMP(converter, symTable, semaCtx, eval, ompNestedLoopCons->value());
+        // Skip OMPD_tile since the tile sizes will be retrieved when
+        // generating the omp.looop_nest op.
```

Comment:
```suggestion
        // Skip OMPD_tile since the tile sizes will be retrieved when
        // generating the omp.loop_nest op.
```

The entire code in `if (optLoopCons.has_value()) {...}` is only intended to raise a `TODO` error?

Reply:
Yes, for right now, but when the lowering is replaced with loop transformations and canonical loops it won't or if other people add nested constructs for some other use they can expand this code.